src/ssmDocument/explorer/documentItemNode.ts

```diff
@@ -49,3 +66,16 @@ export class DocumentItemNode extends AWSTreeNodeBase {
         return await toArrayAsync(this.client.listDocumentVersions(this.documentName))
     }
 }
+
+function fileIconPath(): vscode.ThemeIcon | { light: vscode.Uri; dark: vscode.Uri } {
```

Comment:
This is a copy of the function from `s3FileNode`, correct? I'd say move that function to a separate util file and export. Maybe add a `src/awsexplorer/utils.ts` file?

If you could, also move and export `s3FolderNode`'s `folderIconPath` while you're at it?

Reply:
Will move to` src/awsexplorer/utils.ts`file